Library/Formula/moose.rb

```diff
@@ -0,0 +1,60 @@
+class Moose < Formula
+  desc "MOOSE is a Multiscale Object Oriented Simulation Environment for simulating multiscale neural systems."
+  version "3.0.2"
+  homepage "http://moose.ncbs.res.in"
+  url "https://github.com/BhallaLab/moose-core/archive/ghevar-3.0.2-alpha.zip"
+  sha256 "febc7fc41a2513aeb2b6998731f61c7e7f2a6b5169a77e1612c189779fb0d1f1"
+
+  depends_on "cmake" => :build
+  depends_on "gsl" 
+  depends_on "homebrew/science/hdf5" 
+  depends_on "homebrew/science/libsbml" 
+  depends_on "homebrew/python/matplotlib"
+  depends_on "homebrew/python/numpy"
+  depends_on "python"
+  depends_on "pyqt"
+
+  resource "gui" do
+      version "0.9.0"
+      url "https://github.com/BhallaLab/moose-gui/archive/0.9.0.zip"
+      sha256 "7780bbbc09d754b483c769586c8e566bf83a43408e00ce66c432791fb744aa3a"
```

Comment:
Seeing a different SHA on the test-bot:
```
==> Downloading https://github.com/BhallaLab/moose-gui/archive/0.9.0.zip
==> Downloading from https://codeload.github.com/BhallaLab/moose-gui/zip/0.9.0
Warning: Resource gui reports different SHA256: 7780bbbc09d754b483c769586c8e566bf83a43408e00ce66c432791fb744aa3a
Downloaded to: /Library/Caches/Homebrew/moose--gui-0.9.0.zip
SHA1: f84f5a1d3f38788d23096c88493669d5001e4827
SHA256: f06789f61e48999a3b1d0dc8d25177b8ee6f144610654b120ef2e8b0a204b0a1
```
The version is probably redundant, Homebrew URLs can usually be scanned for `version`.